raycicmd/converter_test.go

```diff
@@ -303,9 +334,15 @@ func TestConvertPipelineStep(t *testing.T) {
 		_, isWait := got["wait"]
 		_, isWanda := test.in["wanda"]
 
-		plugins, ok := got["plugins"]
+		isMac := false
+		jobEnv, ok := test.in["job_env"]
 		if ok {
-			// Check non plugins only.
+			isMac = jobEnv == macosJobEnv
+		}
```

Comment:
nit: just write:

`isMac := ok && jobEnv == macosJobEnv` ?

you can even ignore the `ok` part, as `jobEnv` will be empty string when it is not `ok`, basically:


```
jobEnv, _ := test.in["job_env"]
isMac := jobEnv == macosJobEnv
```